include/wil/resource.h

```diff
@@ -72,6 +72,11 @@ namespace wil
         {
         }
 
+        DWORD get_last_error()
+		{

```

Comment:
Spaces please. And ideally order after the destructor (or at least not sandwiched between two constructors)

Reply:
Agree.  Fix spaces , change location at file, and change name of function to get_saved_error.

Reply:
Agree.  Fix spaces , change location at file, and change name of function to get_saved_error.